api/src/main/java/com/nicknathanjustin/streamercontracts/contracts/ContractsApiController.java

```diff
@@ -192,6 +206,12 @@ public ResponseEntity declineContract(
 
         contractService.setContractState(contractModel, ContractState.DECLINED);
         contractService.settlePayments(contractModel);
+
+        // send notification to proposer
+        final String title = String.format("%s declined your bounty", contractModel.getStreamer().getTwitchUsername());
+        final String description = contractModel.getDescription();
+        alertService.sendNotification(contractModel.getProposer(), title, description);
```

Comment:
code is duplicated here and on lines 151-154... lets pull the duplication into a method